sdk/extensions/Azure.Extensions.AspNetCore.DataProtection.Blobs/tests/DataProtectionTestEnvironment.cs

```diff
@@ -17,18 +17,21 @@ protected override async ValueTask<bool> IsEnvironmentReadyAsync()
         {
             try
             {
-                var client = new BlobServiceClient(BlobStorageEndpoint, Credential);
-                var container = client.GetBlobContainerClient("test");
-                await container.CreateIfNotExistsAsync();
+                for (int i = 0; i < 10; i++)
```

Comment:
Some services use a test-resources-post script that sleeps for a while to allow time for permissions to propagate.

I don't see any task.Delay in here so couldn't this still fail 10 times if executed quickly enough?

Reply:
The test framework does task.Delay after it calls this method.

Reply:
So why do we need to loop 10 times here?

Reply:
It looks like it will already try up to 60 times!

Reply:
>So why do we need to loop 10 times here?

To try and hit different storage servers.

Reply:
ok, may be worth adding a comment mentioning that